assets/javascripts/kitten/index.js

```diff
@@ -118,13 +118,12 @@ export { DocumentIconHouse } from './components/graphics/icons/document-icon-hou
 export { DocumentIconPerson } from './components/graphics/icons/document-icon-person'
 export { DotIcon } from './components/graphics/icons/dot-icon'
 export { DoubleArrowIcon } from './components/graphics/icons/double-arrow-icon'
-export { DownloadIcon } from './components/graphics/icons/download-icon'
+export { DownloadIcon as ExportIcon } from './components/graphics/icons/download-icon'
```

Comment:
Ce qu'on veut c'est avoir `DownloadIcon` et `ExportIcon` dans l'export, sinon c'est un Breaking Change.

```suggestion
export { DownloadIcon as ExportIcon } from './components/graphics/icons/download-icon'
export { DownloadIcon } from './components/graphics/icons/download-icon'
```